core/api.go

```diff
@@ -32,6 +32,22 @@ type clientConfig struct {
 	LogC             *LogConfig
 }
 
+func (config *clientConfig) string() string {
```

Comment:
`string()` -> `String()`, it's typo?

Reply:
no typo, the method is a helper method for sub-struct(`ProducerConfig\PushConsumerConfig`) `String()`